examples/reprojection.js

```diff
@@ -146,18 +146,6 @@ fetch(urlB)
     layers['bng'].setSource(new WMTS(options));
   });
 
-layers['grandcanyon'] = new TileLayer({
```

Comment:
This should also be removed from the dropdown in the html.

Reply:
Thanks for the quick reaction!

I think I did remove it: https://github.com/openlayers/openlayers/pull/11306/files#diff-016b1c92fe0f04f440228e4c4d91b53cL24 -- this should be the only reference to that layer, is it not?

Reply:
Oh sorry @petrsloup, I had overlooked that. Thanks, all good!